image_explorer/image_explorer.py

```diff
@@ -99,20 +99,17 @@ def student_view(self, context):
             if hotspot.feedback.youtube:
                 has_youtube = True
 
-        sprite_url = self.runtime.local_resource_url(self, 'public/images/hotspot-sprite.png')
-
         context = {
             'title': self.display_name,
             'description_html': description,
             'hotspots': hotspots,
             'background': background,
-            'sprite_url': sprite_url,
         }
 
         fragment = Fragment()
         fragment.add_content(loader.render_template('/templates/html/image_explorer.html', context))
-        fragment.add_css(loader.load_unicode('public/css/image_explorer.css'))
-        fragment.add_javascript(loader.load_unicode('public/js/image_explorer.js'))
+        fragment.add_css_url(self.runtime.local_resource_url(self, 'public/css/image_explorer.css'))
+        fragment.add_javascript_url(self.runtime.local_resource_url(self, 'public/js/image_explorer.js'))
```

Comment:
@ziafazal Just to confirm, the changes in L111, L112, and L176 are drive-by changes, right?